tests/src/python/test_provider_oracle.py

```diff
@@ -308,6 +309,40 @@ def testEditPoints(self):
         fid += 1
         self.check_geom(multipoints_z, fid, 'MultiPointZ ((1 2 7),(3 4 8))')
 
+    def testLayerStyles(self):
+        # Need to have explicitly the dbname parameter in the connection string or saveStyle does not work
+        dbconn = "host=localhost dbname=XEPDB1 port=1521 user='QGIS' password='qgis'"
```

Comment:
You have to use self.dbconn instead of this local one because host is oracle and not localhost on CI. That's probably why your tests are failing